python/mxnet/module/executor_group.py

```diff
@@ -558,6 +559,8 @@ def _get_or_reshape(name, shared_data_arrays, arg_shape, arg_type, context, logg
         executor = self.symbol.bind(ctx=context, args=arg_arrays,
                                     args_grad=grad_arrays, aux_states=aux_arrays,
                                     grad_req=self.grad_req, shared_exec=shared_exec)
+        # Get the total bytes allocated for this executor
+        self.total_exec_bytes += int(executor.debug_str().split('\n')[-3].split()[1])
```

Comment:
No, it doesn't. But we definitely can add finer grained profiling stat later (how much extra memory allocated despite shared exec, how it's mapped to ops, which device does it live on, etc)